### PR TITLE
cleanup in chatsharp/handlers/channelhandlers.cs

### PR DESCRIPTION
**Pull request by tparnell8**
issue #7

### CODE REVIEW

File: ChatSharp/Handlers/ChannelHandlers.cs [hunk 1]
```diff
@@ -47,9 +47,9 @@ public static void HandleUserListPart(IrcClient client, IrcMessage message)
         {
             var channel = client.Channels[message.Parameters[2]];
             var users = message.Parameters[3].Split(new[] {' '}, StringSplitOptions.RemoveEmptyEntries);
-            for (int i = 0; i < users.Length; i++)
+            foreach (var user in users)
             {
-                var user = users[i];
+                if(string.IsNullOrWhiteSpace(user))continue;
```
Comment (SirCmpwn) [inline ChatSharp/Handlers/ChannelHandlers.cs]: You're missing some whitespace here. I'll test this and merge it in after celebrations, thanks!

Comment (tparnell8) [inline ChatSharp/Handlers/ChannelHandlers.cs]: do you mean you want it to look like...

`if(string.IsNullOrWhiteSpace(user)) continue;`

the whitespace after the if is not actually required, but I can certainly add it in if you would like

Comment (SirCmpwn) [inline ChatSharp/Handlers/ChannelHandlers.cs]: It's required by the style standards this repository uses :wink: 

And I meant like this: `if (string.IsNullOrWhiteSpace(user)) continue;`

You should always strive to conform to the established style of a codebase you're contributing to.

Comment (tparnell8) [inline ChatSharp/Handlers/ChannelHandlers.cs]: If there are established guidelines you wish enforced, you may want to add it to the development section of readme.md.

In any case I have committed the whitespace fixes you asked for.

**Comment by SirCmpwn** [comment]
Sorry for the delay. I only get notified when you leave a comment, so I didn't know that you had updated the pull request.

**Pull request merged**